geom/src/polygon.rs

```diff
@@ -476,19 +476,38 @@ fn to_geo(pts: &Vec<Pt2D>) -> geo::Polygon<f64> {
     )
 }
 
-fn from_geo(p: geo::Polygon<f64>) -> Polygon {
-    Polygon::buggy_new(
-        p.into_inner()
-            .0
-            .into_points()
+impl From<geo::Polygon<f64>> for Polygon {
```

Comment:
Are you ok with adding the idiomatic conversation traits?  It's a little easier to discover this way since I know where to look rather than custom named methods.

I could add more for other geo types, but only added the minimum of what we needed for now.

Reply:
Yes, these're better! I probably wrote the old thing before I understood `From / Into`